invenio_rdm_records/services/schemas/files.py

```diff
@@ -19,11 +19,12 @@
 class FileSchema(Schema):
     """File schema."""
 
-    checksum = fields.String()
+    id = fields.String(attribute="file.id")
+    checksum = fields.String(attribute="file.checksum")
     ext = fields.String(attribute="file.ext")
     key = SanitizedUnicode()
     metadata = fields.Dict()
-    mimetype = fields.String(attribute="file.mimetype")
+    mimetype = fields.String(attribute="mimetype")
```

Comment:
why not file.mimetype ? seems a bit inconsistent with the rest of the fields

Reply:
The `mimetype` is actually coming from `ObjectVersion.mimetype`, not `ObjectVersion.file.mimetype`, so without this we don't get it at all.

Reply:
maybe makes sense to leave a comment then, someone else will notice the inconsistency for sure. Maybe we are missing a test to cover the absence of this attribute...

Reply:
Actually, the previous value made much more sense... and my logic is not 100% correct based on what I see now in the dumper... I'm double-checking to see if it still works

Reply:
Ok, can confirm it worked with the previous value, because it uses [`RDMFileRecord.file`](https://github.com/inveniosoftware/invenio-records-resources/blob/a6d972fb0f28cebe7c5af1d768b8781a39460238/invenio_records_resources/records/api.py#L77-L80).